wiki/Hit_Objects/en.md

```diff
@@ -1,22 +1,25 @@
+Hit Objects
+===============
+
 A hit object is the core gameplay element in osu!. There are three types of hit objects:
 
--   [Hit Circle](Hit_Circle "wikilink")
--   [Slider](Slider "wikilink")
--   [Spinner](Spinner "wikilink")
+-   [Hit Circle](/wiki/Glossary)
```

Comment:
this term is not defined in the glossary (because I thought they should have their own page)... if need be, I'll add them to my local